codegen/tests/simple/simple_test.go

```diff
@@ -154,7 +154,7 @@ func TestHandlerNotImplemented(t *testing.T) {
 	resp := w.Result()
 	defer resp.Body.Close()
 	body, _ := ioutil.ReadAll(resp.Body)
-	require.JSONEq(t, `{"status":{"code":"1234", "description":"Internal Server Error"}}`, string(body))
+	require.JSONEq(t, `{"status":{"code":"9998", "description":"Internal Server Error"}}`, string(body))
```

Comment:
Any reasons to change it back to 9998?

Reply:
So the tests pass, again, I think there may have been some changes lost from github.service.anz.

Reply:
I believe this was to get the tests to pass, but unclear about the details.

Could you clarify @gkanz ?

Reply:
I've updated all status code to 1234 since Marcelo guesses it might be the internal status codes. If it isn't, that's fine.

Reply:
Ah, okay I think we should probably clarify this... Because it might mean we need to remove the proto files used by that test then.

Reply:
Looks like the error codes are already here: https://github.com/anz-bank/sysl-go/blob/master/common/error_handler.go So the test is just verifying what's already there

Reply:
@ChloePlanet Created an issue to refactor out the error codes in the future https://github.com/anz-bank/sysl-go/issues/19

Otherwise, could you re-review?